tutorial-content/content/reduced-form-specification.md

```diff
@@ -61,65 +61,16 @@ Temperature relationships are often preferred in climate impacts research, becau
 
 ### *Humidity*
 
-There are mainly three metrics for humidity measurements: absolute, relative (often "RH"), and specific. Absolute humidity describes the water content of air,
+There are mainly three metrics for humidity measurements: absolute, relative (often "RH"), and specific (very similar to absolute). Absolute humidity describes the water content of air,
```

Comment:
I would put this in the sentence explaining specific humidity.

Reply:
Done!